server/project.go

```diff
@@ -108,7 +123,8 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 				// We will filter those project with the current principal as an inactive member (the role provider differs from that of the project)
 				roleProvider := project.RoleProvider
 				for _, projectMember := range project.ProjectMemberList {
-					if projectMember.PrincipalID == principalID && projectMember.RoleProvider == roleProvider {
+					if api.Role(projectMember.Role) == api.Owner || api.Role(projectMember.Role) == api.DBA ||
```

Comment:
This is projectMember.Role which is different from workspace role == Owner or DBA.

Reply:
Oh, right. Let me modify it.

Reply:
Actually, we don't need it.